Classes/Utility/Wget/WgetCommand.php

```diff
@@ -518,6 +539,36 @@ public function setNoProxy($noProxy)
         return $this;
     }
 
+    /**
+     * @param boolean $useProxy
+     * @return $this
+     */
+    public function setUseProxy($useProxy)
```

Comment:
Wäre cool, neue Funktionen auch mit type hints zu schrieben

Reply:
The value can either be true (`-e use_proxy=1`), false (`-e use_proxy=0`) or empty/null (option not present in the subprocess call).